samza-core/src/main/java/org/apache/samza/zk/ZkJobCoordinator.java

```diff
@@ -188,6 +193,7 @@ public void onNewJobModelConfirmed(String version) {
   private void generateNewJobModel() {
     // get the current list of processors
     List<String> currentProcessors = zkUtils.getSortedActiveProcessors();
```

Comment:
removing it will requires some more changes, but I guess it will be cleaner. I will add it.